### PR TITLE
feat: updating network resolution data model

### PR DESCRIPTION
**Pull request by harshilgajera-crest**
Added all the values for record_type field as per [IANA website](https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml) for network resolution data model.

### CODE REVIEW

File: pytest_splunk_addon/standard_lib/data_models/Network_Resolution.json [hunk 1]
```diff
@@ -83,10 +83,98 @@
...
+          "AVC",
+          "DOA",
+          "AMTRELAY",
+          "RESINFO",
+          "TA",
+          "DLV",
+          "Private use",
+          "Reserved"
         ],
         "comment": "The DNS resource record type. For details, see the List of DNS record types on Wikipedia."
```
Comment (alexeisuv) [inline pytest_splunk_addon/standard_lib/data_models/Network_Resolution.json:179]: On the [IANA](https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml) web site.

**Review by alexeisuv — CHANGES_REQUESTED** [review]

File: pytest_splunk_addon/standard_lib/data_models/Network_Resolution.json (hunk 1 above)
Comment (harshilgajera-crest) [reply pytest_splunk_addon/standard_lib/data_models/Network_Resolution.json]: Done

**Review by alexeisuv — APPROVED** [review]

**Review by artemrys — APPROVED** [review]

**Pull request merged**